exo/inference/dummy_inference_engine.py

```diff
@@ -0,0 +1,68 @@
+from typing import Optional, Tuple, TYPE_CHECKING
+import numpy as np
+import asyncio
+import json
+from typing import Optional, Tuple
+if TYPE_CHECKING:
+    from exo.inference.inference_engine import InferenceEngine
```

Comment:
What's the purpose of this? Isn't this redundant?

Reply:
Done